tests/unit/models/governing-body-test.js

```diff
@@ -0,0 +1,828 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import sinon from 'sinon';
+import { A } from '@ember/array';
+import { EXECUTIVE_ORGANEN } from 'frontend-organization-portal/models/governing-body-classification-code';
+
+module('Unit | Model | governing body', function (hooks) {
+  setupTest(hooks);
+
+  this.store = function () {
+    return this.owner.lookup('service:store');
+  };
+
+  module('validate', function () {
+    test('it returns errors when model is empty', async function (assert) {
+      const model = this.store().createRecord('governing-body');
+
+      const isValid = await model.validate();
+
+      assert.false(isValid);
+      assert.strictEqual(Object.keys(model.error).length, 2);
+      assert.strictEqual(model.error.startDate.message, 'Vul de startdatum in');
+      assert.propContains(model.error, {
+        endDate: { message: 'Vul de einddatum in' },
+      });
```

Comment:
**nitpick:** What about moving `startDate.message` error in the `propContains` object? ;)

Reply:
Huh, no idea why I did it differently in those two cases.